test/test_operations.py

```diff
@@ -1413,6 +1413,23 @@ def test_fn(t, r):
 
     self.runAtenTest([torch.rand(2, 4), torch.zeros(2, 1)], test_fn)
 
+  def test_spooky_ailing(self):
```

Comment:
lol I can see my name all over the tests now :D 
btw this is essentially `test_pow` in `torch_test_meta.py`.

Reply:
I like that name 😄 Can I keep it? Please ...